src/tests/multihost/alltests/test_proxy_provider_krb_auth.py

```diff
@@ -0,0 +1,126 @@
+""" Automation of proxy provider suite
+
+:requirement: proxy_provider
```

Comment:
Sorry, please update requirement to "IDM-SSSD-REQ : Proxy Provider" and maintain the case

Reply:
Done